docs/datagrid/header.md

```diff
@@ -0,0 +1,74 @@
+# Header options
+
+By default, all columns from the DataTable are imported into the DataGrid in the order they are declared. If you need to exclude certain columns and/or change their order the `header[]` option can then be used to configure this. In addition this option can be used to group headers in a hierarchical structure.
```

Comment:
- Comma after _In addition_
- Suggested rewrite: Use the `header[]` option if you need to exclude certain columns and/or change their order.